### PR TITLE
Register sonom.is-a.dev

### PR DESCRIPTION
**Pull request by getSono**
Register sonom.is-a.dev with CNAME record pointing to getsono.github.io.

### CODE REVIEW

File: domains/sonom.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+        "owner": {
+           "username": "getSono",
+           "email": "julianwolf2013@outlook.de",
+           "discord": "1222250424309121145"
+        },
+    
+        "record": {
+            "CNAME": "getsono.github.io"
```
Comment (DEV-DIBSTER) [inline domains/sonom.json:9]: 404 GitHub page.

**Review by DEV-DIBSTER — CHANGES_REQUESTED** [review]

**Review by triach-rold — CHANGES_REQUESTED** [review]
Website is unreachable. (Github pages 404).

**Comment by getSono** [comment]
> Website is unreachable. (Github pages 404).

I fixed it

**Comment by getSono** [comment]
And please register status.sono.is-a.dev too

**Pull request closed without merging**